
Intro_data = {name :"Glen Joshwa Coelho",
            address: "Mangalore",
            phone : "+91 9901856921",
            email : "glencoelho135@gmail.com",
            github : "https://github.com/biocodeit",
            website : "https://biocodeit.github.io/glencoelho.github.io/",
            linkdin : "https://www.linkedin.com/in/glen-coelho-160aba215",
            summary: `Bioinformatician with a strong foundation in
computational biology and hands-on experience in
molecular simulations. During my time as a Research
Assistant at Quantumzyme, I worked on biocatalysis
projects using tools like GROMACS, AutoDock, Modeller,
and ORCA, building a solid understanding of molecular
modeling and Linux-based workflows.
Recently, I’ve been deepening my skills in Python to build
reusable, modular scripts for automating simulation
pipelines — giving me flexibility to contribute across
scientific computing, bioinformatics, and data-driven
research. I'm passionate about solving problems at the
intersection of biology and code, and I’m always
exploring how better tools can drive better science.`}


// skills
const skills={"Molecular Modeling": ["GROMACS","Umbrella Sampling","AMBER", "g_mmpbsa", "MMPBSA.py", "AutoDock Tools","AutoDock4","Vina","Pymol","VMD","chimera","chimeraX","Modeller","SwissModeller","AlphaFold","CaverDock","Rosetta","Avogadro"],
                "Programming":["Python","HTML","CSS","JavaScript","Bash","Numpy","Pandas","Jupyter Lab","VS code","MDAnalysis","nglview","logging","argparse","Prolif","Multiprocess","MatplotLib","Seaborn","Selenium"],
              "Computational Workflows":["ssh","slurm","bash scripting","micromamaba","conda","miniconda","Git"]}
                
const project_data ={"Project1":{title:"Enzyme Engineering & Molecular Simulation Workflow",
                                 tools:"GROMACS (AMBER/CHARMM force fields), AutoDock Vina, Modeller, VMD, PyMOL, Chimera, Avogadro, MDAnalysis, ORCA",
                                 body :`Worked on computational projects aimed at enhancing enzyme activity using in silico techniques. 
                                 Conducted literature reviews to identify key mutations and functional mechanisms
Built homology models and designed multiple mutants for experimental screening
Performed post-MD trajectory analysis including RMSD, RMSF, SASA, distance metrics, and MM/PBSA for
protein-ligand complexes
Executed umbrella sampling and metadynamics studies for free energy profiling
Experienced with rigid and flexible docking using AutoDock Vina
Used GROMACS with both AMBER and CHARMM force fields across multiple simulation campaigns
Strong practical experience with molecular visualization and editing using VMD, PyMOL, Chimera, and Avogadro` 
},
"Project2":{title:"Python Automation for Molecular Dynamics System Setup",
                                 tools:"Python, GROMACS, MDAnalysis, Modeller, ParmEd, Bash, argparse, logging, JupyterLab, Git, Pandas, Numpy, Nglview, Prolif, Pymol",
                                 body :`Developed Python-based command-line tools to automate the complete setup of molecular dynamics
systems, including protein–ligand preparation, mutant modeling, topology merging, solvation, and ion
addition.
Designed modular scripts with log-based checkpointing, input file parsing, and batch execution
support for handling multiple variants efficiently.
Integrated key scientific libraries such as MDAnalysis, Modeller, and ParmEd with GROMACS workflows,
emphasizing reproducibility, scalability, and automation in structural biology pipelines.<a href = "https://github.com/biocodeit/MD_WFlow">(https://github.com/biocodeit/MD_WFlow)</a>` 
},
"Project3":{title:"Python Automation for Trajectory Analysis and MM/PBSA Calculations",
                                 tools:"Python, GROMACS, MDAnalysis, argparse, pandas, bash, custom modules, Git",
                                 body :`Developed a modular Python script to automate post-MD simulation analysis, including RMSD, RMSF,
distance monitoring, SASA, hydrogen bonding, residue contacts, interaction energy, and MM/PBSA
free energy calculations.
Implemented flexible input parsing, auto-checkpointing, and directory handling to streamline multifeature analysis using MDAnalysis and GROMACS outputs.
Enabled batch execution of complex analyses through structured input files and reusable
components, improving reproducibility and reducing manual effort.<a href = "https://github.com/biocodeit/my_MD">(https://github.com/biocodeit/my_MD)</a>` 
},
"Project4":{title:"Amino Acid Guesser- Interactively Learning Aminio Acids ",
                                 tools:"HTML, CSS, JavaScript, GitHub, VS code",
                                 body :`Built an interactive web game to help users
                                memorize and learn amino acids through a dynamic, quiz-style interface. Designed and implemented the game logic using 
                                vanilla JavaScript, including random option generation for each round to maximize learning engagement. Integrated 
                                3Dmol.js for real-time 3D visualization of amino acid structures and incorporated SVG images for clear, scalable visuals. 
                                Deployed the site via Netlify, ensuring easy access and sharing. Focused on blending educational value with gamified 
                                features to support biochemistry students and instructors.<a href = "https://biocodeit.github.io/amino-guesser/">(https://biocodeit.github.io/amino-guesser/)</a>` 
},
"Project5":{title:"Resume Builder- Builds a simple Proffesional Resume ",
                                 tools:"HTML, CSS, JavaScript, GitHub, VS code",
                                 body :`Developed a web-based tool to generate professional resumes using user-provided JSON data as input. Utilized 
                                 vanilla JavaScript for dynamic content rendering and scalability, and applied modern CSS techniques for clean, 
                                 responsive styling. The application streamlines resume creation for personal websites or job applications, supporting 
                                 customization and quick updates.<a href = "https://github.com/biocodeit/build-resume">(https://github.com/biocodeit/build-resume)</a>` 
}}

edu_data={edu1:
          {degree : "M.Sc",
          subject : "Biotechnology",
          start : "2021",
          end : "2023",
          university :"Mangalore University, Manglaore"},
        edu2: {
          degree : "B.Sc",
          subject : "Biotechnology",
          start : "2018",
          end : "2021",
          university :"St. Aloysius College, Manglaore"
        }}

exp_data= {exp1 :
          {role:"Reserach Assistant",
          company : "Quantumzyme",
          start : "Aug 2024",
          end : "Present",
          body: `Part of a team involved in engineering enzymes to
improve their efficiency using computational
techniques for industrial biocatalysis, where I used
MD simulations, homology modeling, and docking.`
          },
        exp233 :
          {role:"Internship",
          company : "Quantumzyme",
          start : "Feb 2024",
          end : "July 2024",
          body: `Learnt in silico techniques like molecular docking,
homology modeling, molecular visualization, MD,
and QM. Also improved my presentation skills and
actively contributed to live research projects.`
          }}

achieve_data = ["Ranked 440 in GATE BT 2023",
                "Qualified TIFR/JGEEBILS Exam",
                "Scored 85 percentile in CSIR-NET"]

languages= ["English","Hindi","Kannada"]

